docs/api.rst

```diff
@@ -0,0 +1,1361 @@
+
+DELETE an external system
+-------------------------
+**URL**:
+
+.. http:delete:: api/externalsystem/id/(int: externalsystem_id)
+
+**Example Request**:
+
+.. sourcecode:: http
+
+    DELETE /api/externalsystem/id/3/
+    Host: example.com
+    Accept: application/json
+    Api-token:
+
+**Example Response**:
+
+.. sourcecode:: http
+
+    HTTP/1.1 204 OK
+    Vary: Accept
+    Content-Type: application/json
+
+
+External Record
+===============
+
+GET an external record
+----------------------
+**URL**:
+
+.. http:get:: api/externalrecord/id/(int: externalrecord_id)/
+
+**Example Request**:
+
+.. sourcecode:: http
+
+    GET /api/externalrecord/id/27871
+    Host: example.com
+    Accept: application/json
+    Api-token:
+
+**Example Response**:
+
+.. sourcecode:: http
+
+      HTTP/1.1 200 OK
+      Vary: Accept
+      Content-Type: application/json
+
+      {
+      "created": "2018-06-04 16:47:40.320305",
+      "modified": "2018-06-04 16:47:40.320347",
+      "label": 1,
+      "record_id": "QLUBPG4Y0U8Y67TZ:JIEEDIOEP",
+      "path": "CBTTC - Specimen Only",
+      "external_system": 2,
+      "id": 27871,
+      "subject": 4921
+      }
+
+GET an external record with label
+---------------------------------
+**URL**:
+
+.. http:get:: /api/externalrecord/labels/(int: externalrecordlabel_id)/
+
+**Example Request**:
+
+.. sourcecode:: http
+
+      GET /api/externalrecord/labels/82/
+      Host: example.com
+      Accept: application/json
+      Api-token:
+
+**Example Response**:
+
+.. sourcecode:: http
+
+      {
+      "id": 82,
+      "label": "This is a test"
+      }
+
+
+POST (query) to obtain external record info
+-------------------------------------------
+**URL**:
+
+.. http:post:: /api/externalrecord/query/
+
+**Example Request**:
+
+.. sourcecode:: http
+
+      Host: example.com
+      POST: /api/externalrecord/query/
+      Content-Type: application/json
+      Api-token:
+      Body:
+      [
+      {
+        "subject_id":"2",
+        "external_system_id":"2",
+        "path":"Test Protocol"
+      },
+      {
+        "subject_org":,
+        "subject_org_id"
+      },
+      {
+        "subject_id":,
+        "external_system_name":,
+      },
+      {
+        "subect_id":,
+        "external_system_url":
+      }
+      ]
+
+**Example Response**:
+
+.. sourcecode:: http
+
+    HTTP/1.1 200 OK
+    Vary: Accept
+    Content-Type: application/json
+
+    [
+    {
+        "external_record": [
+            {
+                "created": "2014-01-28 13:42:41.693000",
+                "modified": "2014-01-28 13:42:41.693000",
+                "label": 1,
+                "record_id": "NXB546EUZSDLZKGR:5EM3AOORG",
+                "path": "Test Protocol",
+                "external_system": 2,
+                "id": 1,
+                "subject": 2
+            }
+        ],
+        "path": "Test Protocol",
+        "subject_id": "2",
+        "external_system_id": "2"
+    }
+]
+
+POST to obtain create an external Record
```

Comment:
why do you have 'obtain and create' here?

Reply:
73ede78580a4bc68c23235a769e476bb1b6fee36